From 66a11945dafd2680a158abc4f268481b0a64f55d Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 16 Nov 2017 14:56:05 +0800
Subject: [PATCH] ARM: dts: rk312x-android: add "shared-dma-pool" for cma
 region

Without the "shared-dma-pool" compatible, the region reserved is
not an dma region.

Support system-heap.

Change-Id: I25aa3df38db2522bf3e580e96ed6e0f5ecd4f5fe
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk312x-android.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/rk312x-android.dtsi b/arch/arm/boot/dts/rk312x-android.dtsi
index 2bc74c7f714f..ade77b9080b2 100644
--- a/arch/arm/boot/dts/rk312x-android.dtsi
+++ b/arch/arm/boot/dts/rk312x-android.dtsi
@@ -124,7 +124,10 @@
 		ranges;
 
 		cma_region: region@6a000000 {
+			compatible = "shared-dma-pool";
+			no-map;
 			reg = <0x6a000000 0x1000000>;
+			linux,cma-default;
 		};
 
 		ramoops_mem: ramoops@68000000 {
@@ -146,6 +149,8 @@
 
 		cma-heap {
 		};
+		system-heap {
+		};
 	};
 
 	ramoops {
-- 
2.35.3

